src/executor/evaluate/mod.rs

```diff
@@ -281,5 +283,53 @@ async fn evaluate_function<'a, T: 'static + Debug>(
 
             Ok(Evaluated::from(Value::Str(converted)))
         }
+        Function::Ceil(expr) => {
+            let number = match eval(expr).await?.try_into()? {
+                Value::F64(number) => Some(number),
+                Value::Str(s) => match f64::from_str(&s) {
+                    Ok(f) => Some(f),
+                    Err(_) => None,
+                },
+                Value::I64(number) => f64::from_i64(number),
+                _ => None,
+            };
+
+            match number {
+                Some(number) => Ok(Evaluated::from(Value::F64(number.ceil()))),
+                None => Err(EvaluateError::FunctionRequiresFloatValue("CEIL".to_owned()).into()),
+            }
+        }
+        Function::Round(expr) => {
+            let number = match eval(expr).await?.try_into()? {
+                Value::F64(number) => Some(number),
+                Value::Str(s) => match f64::from_str(&s) {
+                    Ok(f) => Some(f),
+                    Err(_) => None,
+                },
+                Value::I64(number) => f64::from_i64(number),
+                _ => None,
+            };
+
+            match number {
+                Some(number) => Ok(Evaluated::from(Value::F64(number.round()))),
+                None => Err(EvaluateError::FunctionRequiresFloatValue("ROUND".to_owned()).into()),
+            }
+        }
```

Comment:
Good start! I think you also recognize there are some redundant codes which could be wrapped by a function.
Let's reduce redundancy in `Ceil`, `Round` and `Floor`.
Not just less redundancy helps programmers to write less codes, but also it is better because there's also less stuffs to be tested.